torch_xla/experimental/pallas_kernels/quantized_matmul_kernel.py

```diff
@@ -216,6 +205,29 @@ def quantized_matmul_int8(
   # For best performance, only enable this behavior when single input block is used per batch.
   save_q_x = quantize_activation and n_in == 1 and n_out > 1
 
+  acc_dtype = jnp.int32 if quantize_activation else jnp.float32
+
+  # Calculate in/out VMEM size
+  vmem_in_out = (
+      batch_block_size * in_block_size * x.dtype.itemsize +
+      out_block_size * in_block_size * w.dtype.itemsize +
+      out_block_size * scalar.dtype.itemsize +
+      batch_block_size * x_abs_max_val.dtype.itemsize +
+      batch_block_size * out_block_size * x.dtype.itemsize)
+  # Calculate scratch VMEM size
+  vmem_scratch = 0
+  if save_acc:
+    vmem_scratch += batch_block_size * out_block_size * 4  # acc_scratch
```

Comment:
could you replace 4 with "acc_dtype"? If acc_dtype is not int32, then it'll not be 4.

Reply:
See below comment for more info.